st2common/st2common/transport/consumers.py

```diff
@@ -43,6 +43,7 @@ def __init__(self, connection, queues, handler):
         self._handler = handler
 
     def shutdown(self):
+        self.should_stop = True
```

Comment:
What is this for?

Reply:
This will shutdown the WFE queue consumer, so that the WFE doesn't pick new requests from the queue. This was added for actionrunner sometime back. https://github.com/celery/kombu/blob/master/kombu/mixins.py#L131